server/team_mgmt/api_test.py

```diff
@@ -0,0 +1,79 @@
+import requests
```

Comment:
These tests don't seem to assert anything - just print out the responses. Can you update these.